apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/UnclaimedActivityRecordIntegrationTest.scala

```diff
@@ -0,0 +1,311 @@
+              None,
+            )
+          }
+        },
+      )("vote request and UnallocatedUnclaimedActivityRecord have been created",
+        _ => {
+          sv1Backend.listVoteRequests().loneElement
+          eventually(){
+            sv1Backend.participantClient.ledger_api_extensions.acs
+              .filterJava(UnallocatedUnclaimedActivityRecord.COMPANION)(dsoParty) should not be empty
+          }
+        }
+      )
+    }
+
+    setTriggersWithin(
+      triggersToPauseAtStart = expiredUnclaimedTriggers :+ aliceMergeAmuletsTrigger,
+      triggersToResumeAtStart = Seq.empty,
+    ) {
+      clue("UnclaimedActivityRecord has been created"){
+        eventually(){
+          sv1Backend.participantClient.ledger_api_extensions.acs
+            .filterJava(UnclaimedActivityRecord.COMPANION)(dsoParty) should not be empty
+
+          sv1Backend.participantClient.ledger_api_extensions.acs
+            .filterJava(UnclaimedReward.COMPANION)(dsoParty) shouldBe empty
+        }
+      }
+    }
+
+    setTriggersWithin(
+      triggersToPauseAtStart = Seq(aliceMergeAmuletsTrigger),
+      triggersToResumeAtStart = Seq.empty,
+    ) {
+      clue("UnclaimedActivityRecord gets archived"){
+        eventually(){
+          sv1Backend.participantClient.ledger_api_extensions.acs
+            .filterJava(UnclaimedActivityRecord.COMPANION)(dsoParty) shouldBe empty
+        }
+      }
+    }
+  }
+
+  "Amulet gets minted" in { implicit env =>
+    val sv1UserId = sv1WalletClient.config.ledgerApiUser
+    val sv1Party = sv1Backend.getDsoInfo().svParty
+    val aliceUserName = aliceWalletClient.config.ledgerApiUser
+    val aliceParty = aliceValidatorBackend.onboardUser(aliceUserName)
+    val unclaimedRewardsToMint = Seq(12.0, 2.0, 5.0)
+    val amountToMint = 15.0
+
+    def allocateTriggers: Seq[Trigger] =
+     activeSvs.map(_.dsoDelegateBasedAutomation.trigger[AllocateUnallocatedUnclaimedActivityRecordTrigger])
+    // The trigger that merges amulets for alice
+    // Note: using `def`, as the trigger may be destroyed and recreated (when user is offboarded and onboarded)
+    def aliceMergeAmuletsTrigger: Trigger =
+     aliceValidatorBackend
+       .userWalletAutomation(aliceUserName)
+       .futureValue
+       .trigger[CollectRewardsAndMergeAmuletsTrigger]
+
+    aliceWalletClient.list().amulets should have length 0
+
+    actAndCheck(
+     "Mint some unclaimed rewards",
+     unclaimedRewardsToMint.foreach { amount =>
+       createUnclaimedReward(
+         sv1ValidatorBackend.participantClientWithAdminToken,
+         sv1UserId,
+         amount
+       )
+     },
+    )(
+      "Unclaimed rewards gets created",
+      _ => {
+        sv1Backend.participantClient.ledger_api_extensions.acs
+          .filterJava(UnclaimedReward.COMPANION)(dsoParty) should not be empty
+      },
+    )
+
+    setTriggersWithin(
+     triggersToPauseAtStart = allocateTriggers,
```

Comment:
similar to above, I seen no reason to pause anything. The intermediate steps just make the test slower.